libs/wingc/src/parser.rs

```diff
@@ -977,8 +977,41 @@ impl<'s> Parser<'s> {
 			};
 		}
 
+		if WINGSDK_BRINGABLE_MODULES.contains(&module_name.name.as_str()) || module_name.name == WINGSDK_STD_MODULE {
+			return Ok(StmtKind::Bring {
+				source: BringSource::BuiltinModule(module_name),
+				identifier: alias,
+			});
+		}
+
+		// check if a trusted library exists with this name
+		let source_dir = Utf8Path::new(&self.source_name).parent().unwrap();
+		let module_dir =
+			wingii::util::package_json::find_dependency_directory(&format!("@winglibs/{}", module_name.name), &source_dir)
+				.ok_or_else(|| {
+					self
+						.with_error::<Node>(
+							format!(
+								"Could not find a trusted library \"@winglibs/{}\" installed. Did you mean to run `npm i @winglibs/{} --save`?",
```

Comment:
```suggestion
								"Could not find a trusted library \"@winglibs/{}\" installed. Did you mean to run `npm i @winglibs/{}`?",
```

nit: I believe `--save` is the default